bigframes/core/compile/aggregate_compiler.py

```diff
@@ -21,31 +21,63 @@
 import pandas as pd
 
 import bigframes.constants as constants
+import bigframes.core.compile.scalar_op_compiler as scalar_compilers
+import bigframes.core.expression as ex
 import bigframes.core.window_spec as window_spec
 import bigframes.dtypes as dtypes
 import bigframes.operations.aggregations as agg_ops
 import third_party.bigframes_vendored.ibis.expr.operations as vendored_ibis_ops
 
+scalar_compiler = scalar_compilers.scalar_op_compiler
 
-def compile_unary_aggregate(
-    op: agg_ops.AggregateOp, input: ibis_types.Column
+
+def compile_aggregate(
+    aggregate: ex.Aggregation,
+    bindings: typing.Dict[str, ibis_types.Value],
 ) -> ibis_types.Value:
-    return compile_agg(op, input)
+    if isinstance(aggregate, ex.UnaryAggregation):
```

Comment:
Note to self: Is there another class for 0 argument aggregations (e.g. row count) that we need to handle here?

Reply:
CL title is a lie, I haven't actually defined the 0-arg aggregation ops yet. They still pretend to be 1-arg aggregations. Will convert 0-arg ops (rank is another one) in a follow-up pr.